batch-processing-massive-data-much-quicker-with-tispark.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Batch Processing Massive Data Much Quicker with TiSpark
+author: ['Zhexuan Yang']
+date: 2020-12-22
+summary: A senior solution architect at PingCAP explains why TiSpark (TiDB + Apache Spark) is better than the traditional batch processing solution, how you can benefit from TiSpark, and how it works.
+tags: ['TiSpark']
+categories: ['Engineering']
+image: /images/blog/batch-processing-massive-data-much-quicker-with-tispark.jpg
+aliases: ['/blog/batch-processing-massive-data-much-quicker-with-tispark']
```

Comment:
Why do you add an alias?

Reply:
I copied it from another blog. Can you add the user guide for the aliases in our blog writing tutorial?